Core/Base/AgentModule.py

```diff
@@ -20,13 +18,16 @@
 from DIRAC.Core.Utilities.ReturnValues import isReturnStructure
 from DIRAC.Core.Utilities import Time, MemStat
 
+__RCSID__ = "$Id$"
+
+
 def _checkDir( path ):
   try:
     os.makedirs( path )
-  except Exception:
+  except OSError:
```

Comment:
I have added a `mkDir` utility that works like "mkdir -p " and use it in many, not all, modules. Please check.